platform-hub-api/Dockerfile

```diff
@@ -1,15 +1,14 @@
-FROM alpine:3.5
+FROM ruby:2.3.8-alpine
 
 ENV RAILS_ENV production
 
 RUN apk update && apk upgrade \
     && apk --update add ca-certificates libressl \
     && update-ca-certificates \
-    && apk --update add \
-    bash curl ruby ruby-irb ruby-rake ruby-io-console ruby-json ruby-bigdecimal libstdc++ tzdata \
-    postgresql-client \
+    && apk --update add bash curl postgresql-client postgresql-dev \
+    && apk -U add --virtual build_deps build-base make \
     && echo 'gem: --no-document' > /etc/gemrc \
-    && gem install bundler -v 1.16.2
+    && gem install bundler -v 2.0.2
```

Comment:
I recall Bundler v2+ having some issues, so may be worth sticking with `1.16.2`? Also, the `Gemfile.lock` has been explicitly built with `1.16.2`, which will only cause a warning when `bundle install` is run, but may as well stick to this version unless there are security patches for it?

Reply:
There wasn't anything that came up for bundler from what I recall so will revert it back 👍